Realm.Shared/native/Configuration.cs

```diff
@@ -26,6 +27,7 @@ namespace Realms.Native
     internal delegate bool MigrationCallback(IntPtr oldRealm, IntPtr newRealm, Schema oldSchema, ulong schemaVersion, IntPtr managedMigrationHandle);
 
     [StructLayout(LayoutKind.Sequential)]
+    [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter")]
```

Comment:
Would it be possible to apply this rule to everything in the `Realms.Native` namespace perhaps?